src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -1996,7 +2128,16 @@ namespace Google.Example.Library.V1.Samples
                 SeriesUuid = new SeriesUuid(),
             };
             PublishSeriesResponse response = libraryServiceClient.PublishSeries(request);
-            // FIXME: inspect the results
+            // testing writing bytes field.
+            using (Stream output = File.Create($"uuid_of_series_with_book_{response.BookNames[0]}.raw"))
+            {
+                response.SeriesUuid.SeriesBytes.writeTo(output);
```

Comment:
If you wish to use this form of writing, then the method `WriteTo()` has a capital `W`.